_src/static/sass/main-rtl.scss

```diff
@@ -13,14 +13,19 @@
 @import '../../vendor/bi-app/bi-app-rtl';   // RTL support
```

Comment:
I moved BASE above UTILITIES because we're including `_variables.scss` in our BASE set, which should be compiled before any utilities that reference them. Is this incorrect?